extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/__init__.py

```diff
@@ -1,7 +1,9 @@
 from .local_inference.text_2_image import HuggingFaceText2ImageDiffusor
 from .local_inference.text_generation import HuggingFaceTextGenerationTransformer
-from .remote_inference_client.text_generation import HuggingFaceTextGenerationClient
+from .local_inference.text_summarization import HuggingFaceTextSummarizationTransformer
 
-LOCAL_INFERENCE_CLASSES = ["HuggingFaceText2ImageDiffusor", "HuggingFaceTextGenerationTransformer"]
+# from .remote_inference_client.text_generation import HuggingFaceTextGenerationClient
```

Comment:
We should keep this line. It's used in the `remote_inference` part of this extension

Reply:
This errors for me FYI

Reply:
Huh weird, ok can you pls file issue for this and assign to me to followup?